lerna.json

```diff
@@ -1,6 +1,5 @@
 {
   "lerna": "2.10.1",
-  "packages": ["packages/*"],
```

Comment:
Why remove this? Is it the default?

Reply:
Same concern. I know that is the default set it in the init, but not that works without any package initialisation.
Basically this is what allow us to have `-dev` versions

Reply:
Since yarn workspaces are used It's obsolete when `"useWorkspaces": true,` is set since lerna 2.0.

https://classic.yarnpkg.com/blog/2017/08/02/introducing-workspaces/

```
Starting with Lerna 2.0.0, when you pass the flag --use-workspaces when running Lerna commands, it will use Yarn to bootstrap the project and also it will use package.json/workspaces field to find the packages instead of lerna.json/packages.
```

Reply:
At some point you have to decide to either use the config of yarn or the config of lerna. Right now both are used. This change removed the lerna config now.